articles/active-directory/develop/quickstart-v2-java-webapp.md

```diff
@@ -102,17 +114,31 @@ To run this sample you will need:
 > - `Enter_the_Client_Secret_Here` - is the **Client Secret** you created in **Certificates & Secrets** for the application you registered.
 
 #### Step 4: Run the code sample
-1. Run the code sample, and open a browser and navigate to *http://localhost:8080*.
-1. The front page contains a **sign-in** button. Click on the **sign-in** button to redirect to Azure Active Directory. The user will be prompted for their credentials.  
-1. After successfully authenticating on Azure Active Directory, they will be redirected to *http://localhost:8080/msal4jsamples/secure/aad*. They are officially signed in to the application, and the page should show information for the signed in account. It will also contain buttons for: 
-    - *Sign Out*: Will sign out the current user from the application, and redirect them the home page.
-    - *Show Users*: Will acquire a token for the Microsoft Graph, then call the Microsoft Graph with the token attached to the request to get all of the users in the tenant.
+
+To run the project, you can either:
+
+Run it directly from your IDE by using the embedded spring boot server or package it to a WAR file using [maven](https://maven.apache.org/plugins/maven-war-plugin/usage.html) and deploy it to a J2EE container solution such as [Apache Tomcat](http://tomcat.apache.org/).
+
+##### Running from IDE
+
+If you are running the web application from an IDE, click on run, then navigate to the home page of the project. For this sample, the standard home page URL is <http://localhost:8080>
```

Comment:
What are the angle brackets for?
I'd put the inline code marker around this, e.g `http://localhost:8080`

Reply:
@TylerMSFT, @sangonzal  FYI - Ramya can add these feedbacks in her final commit. But otherwise, we want to close and merge this PR asap since we need the initial changes with the code snippets for portal integration by tomorrow. 
Any further edits of the wording on this doc can be made in a follow up PR later.

Reply:
fine.